packages/builders/scripts/fast-usdc/init-fast-usdc.js

```diff
@@ -17,12 +19,35 @@ import { parseArgs } from 'node:util';
  * @import {CoreEvalBuilder, DeployScriptFunction} from '@agoric/deploy-script-support/src/externalTypes.js'
  * @import {ParseArgsConfig} from 'node:util'
  * @import {FastUSDCConfig} from '@agoric/fast-usdc/src/fast-usdc.start.js'
+ * @import {Passable} from '@endo/marshal';
+ * @import {CosmosChainInfo} from '@agoric/orchestration';
  */
 
 const { keys } = Object;
 
+const defaultAssetInfo = {
+  uusdc: {
+    baseName: 'noble',
+    chainName: 'noble',
+    baseDenom: 'uusdc',
+  },
+  [`ibc/${denomHash({ denom: 'uusdc', channelId: fetchedChainInfo.agoric.connections['noble-1'].transferChannel.channelId })}`]:
+    {
+      baseName: 'noble',
+      chainName: 'agoric',
+      baseDenom: 'uusdc',
+      brandKey: 'USDC',
```

Comment:
This is good feedback. I went with `brandKey` since I'm not well versed in this area. 

I created a ticket to track the removal of `brandKey` in favor of this approach: https://github.com/Agoric/agoric-sdk/issues/10580

Reply:
Thanks. I think it's an entirely internal interface, so it's not a big deal.